Makefile

```
@@ -10,6 +10,7 @@ BUILD_FLAGS = -mod=readonly -ldflags "$(LD_FLAGS)"
 HTTPS_GIT := https://github.com/cometbft/cometbft.git
 CGO_ENABLED ?= 0
 
+BUF_VERSION := v1.29.0
```

Thanks! Out of curiosity, why v1.29.0 instead of latest release? 
```suggestion
BUF_VERSION := v1.31.0
```

Ref: https://github.com/bufbuild/buf/releases/tag/v1.31.0

---

Yeah I use the version mentioned [here](https://github.com/celestiaorg/celestia-core/pull/1249#discussion_r1508274913), I guess using latest release is better.

---

Oh I see, yea if latest doesn't cause any issues then let's bump to latest.